contact.html

```diff
@@ -52,17 +52,22 @@
         </h1>
       <div class="field">
         <label>Name</label>
-        <input type="text" name="name" placeholder="Name">
+        <input type="text" name="name" placeholder="Name" required />
       </div>
       <div class="field">
         <label>E-mail</label>
-        <input type="text" name="email" placeholder="E-mail">
+        <input type="text" name="email" placeholder="E-mail" required />
+      </div>
+      <div class="field">
+        <label>Subject</label>
+        <input type="text" name="_subject" placeholder="Subject" required />
```

Comment:
Do we want custom subjects for should we have a few options for them in a drop-down?